### DISCUSSION TITLE
timer setting

### DISCUSSION THREAD

**Discussion by Troubadix** [arendst/Tasmota#22278] [Q&A]
Hi,

I'm using Delock 11828 USB switch running Tasmota 14.2.0 for powering a USB-powered stepping motor.

I want the device to turn on for 2 seconds, then turn it off for 40 minutes and then start again with 2 sec/40 min/2 sec/40 min/....

How can I achieve this?

Thank you!

**Comment by sfromis** [comment]
Something like rules:
```
on Power1#state=0 do RuleTimer1 2400 endon 
on Rules#timer=1 do TimedPower1 2000,on endon
```
`RuleTimer` is seconds, and `TimedPower` is reverting the action after milliseconds. The cycle starts after first power-off (or `RuleTimer1` command) by "other means".

**Comment by Troubadix** [comment]
Sadly this doesnt work
`10:14:41.871 CMD: on Power1#state=0 do RuleTimer1 2400 endon  on Rules#timer=1 do TimedPower1 2000,on endon
10:14:41.877 RSL: RESULT = {"Command":"Unknown"}`

**Reply by sfromis** [reply]
You need to use the `Rule` command to create/update a rule set, and to enable/disable it.
